server/views/components/input/input.njk

```diff
@@ -6,7 +6,7 @@
     value="{{ model.value }}"
     placeholder="{{ model.placeholder }}"
     {{ 'disabled' if model.disabled }}
-    {{ 'autofocus' if model.autofocus  }} />
+    {{ 'autofocus' if model.autofocus and not model.value }} />
```

Comment:
I guess this means this will be true for _all_ future autofocus-able inputs – I think I'm OK with that.